### PR TITLE
Do not assume action parameters are strings

### PR DESCRIPTION
**Pull request by szschaler**
This PR fixes a bug where the platform crashed when an action parameter wasn't actually a string

### CODE REVIEW

**Review by szschaler — COMMENTED** [review]
This is important for https://github.com/mdenet/platformtools/pull/53 to work

File: platform/src/Playground.js [hunk 1]
```diff
@@ -872,7 +872,7 @@ function runAction(source, sourceButton) {
         } else {
             // No panel with ID so it use as the parameter value
             const parameterValue = action.parameters[paramName];
-            console.log(`The panel ID starting with ${parameterValue.substring(0,10)}... not found. Using it as a the parameter value for ${paramName}.`);
+            // console.log(`The panel ID starting with ${parameterValue.substring(0,10)}... not found. Using it as a the parameter value for ${paramName}.`);
```
Comment (barnettwilliam) [inline platform/src/Playground.js:875]: Remove or fix, commenting it out it'll just be forgotten why it there.

**Review by barnettwilliam — CHANGES_REQUESTED** [review]
Commented out code

**Review by barnettwilliam — APPROVED** [review]
LGTM

**Pull request merged**